components/Step2/Step2.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {Image, Text, View} from 'react-native';
+import RadioButton from '../RadioButton/RadioButton';
+import constants from '../../utils/constants';
+import {updateUserData} from '../../redux/reducers/WizardData';
+import {useDispatch, useSelector} from 'react-redux';
+
+import globalStyle from '../../assets/styles/globalStyle';
+
+export const Step2 = () => {
+  const dispatch = useDispatch();
+  const wizardData = useSelector(
+    (state: {wizard: {level: string}}) => state.wizard,
```

Comment:
All of your selectors should have selector functions for reusability purposes. Ex: https://redux.js.org/usage/deriving-data-selectors#define-selectors-alongside-reducers